packages/react/src/components/OverflowMenu/OverflowMenu.js

```diff
@@ -564,20 +560,20 @@ class OverflowMenu extends Component {
 
     return (
       <ClickListener onClickOutside={this.handleClickOutside}>
-        <button
+        <IconButton
           {...other}
           type="button"
           aria-haspopup
           aria-expanded={this.state.open}
           className={overflowMenuClasses}
           onKeyDown={this.handleKeyPress}
           onClick={this.handleClick}
-          aria-label={ariaLabel}
```

Comment:
Does this mean we can also remove this from the props destructure at the top of `render()`?

Reply:
so we're still using this prop to pass onto the `UL` 
<img width="419" alt="image" src="https://user-images.githubusercontent.com/40550942/214405252-5f24b590-0367-4100-bcab-21b9d6b4b323.png">

So I'd say we still need to destructure it?